SixTrack/sixtrack.s

```diff
@@ -50086,6 +50096,7 @@ c$$$            endif
      &epsnxyzv ! normalized emittances
       double precision :: tunelask,tuneffti,tunefft,tuneapa,tunefit,    &
      &tunenewt,tuneabt2,tuneabt,tunenewt1 !Define the functions to be called from PLATO
+      double precision :: dump_last_turn ! auxiliary variable for loop over turns
```

Comment:
Thou shall not use "double" variables as loop indices. I'm changing it to an int. This even made the compiler scream...